### PR TITLE
httplib2 - v0.22.0 ❄️

### PR DESCRIPTION
**Pull request by Andrii-Osipov**
httplib2 - v0.22.0 :snowflake:

**Destination channel:** Snowflake

- [PKG-6071](https://anaconda.atlassian.net/browse/PKG-6071) 
- [httplib2](https://github.com/httplib2/httplib2)
- [PyPi](https://pypi.org/project/httplib2/)
- [conda-forge](https://github.com/conda-forge/httplib2-feedstock)

[PKG-6071]: https://anaconda.atlassian.net/browse/PKG-6071?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,42 +1,81 @@
...
     - pip
-    - pytest
+    - setuptools >=40.8.0
+    - wheel
   run:
-    - python >=3
+    - python
     - pyparsing >=2.4.2,<3
 
+# Wont fix. Needs reastimation.
```
Comment (scotthain) [inline recipe/meta.yaml:27]: Some of these tests seem like they could expose security risks or unexpected behavior, I'm assuming that's a risk we're willing to take on? (for the record, I don't know anything about these tests, but httplib2 feels... important)

Comment (Andrii-Osipov) [reply recipe/meta.yaml:27]: @scotthain 
This lib is used as a dependency for google-auth-httplib2.
And google-auth-httplib2 tests work without issues.
CC: @lorepirri

Comment (scotthain) [reply recipe/meta.yaml:27]: As long as we have adequate coverage, I'm 👍

**Review by scotthain — APPROVED** [review]
🚀 🍰

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,42 +1,81 @@
...
-    - python >=3
+    - python
     - pip
-    - pytest
+    - setuptools >=40.8.0
+    - wheel
   run:
-    - python >=3
+    - python
     - pyparsing >=2.4.2,<3
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - pyparsing >=2.4.2,<4
```
https://github.com/httplib2/httplib2/blob/v0.22.0/requirements.txt#L2

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,42 +1,81 @@
...
     - pip check
+    - pytest -v --ignore=tests/_TODO_legacy/python2 {{ deselect_tests }} tests
   requires:
     - pip
+    - pytest
+    - cryptography
+    - six
+    - nose
+    - responses
+    - flake8
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
You don't seem to be running flake8

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,42 +1,81 @@
...
+    - future
+    - pytest-cov
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
Not running coverage either.

**Review by cbouss — APPROVED** [review]

**Review by lorepirri — APPROVED** [review]

**Pull request merged**